content/influxdb/v2.0/tools/google-data-studio.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Use the Google Data Studio connector
+description:
+menu:
+  influxdb_2_0:
+    parent: Tools & integrations
+weight: 102
+influxdb/v2.0/tags: [google]
+---
+
+The [InfluxDB Google Data Studio connector](https://datastudio.google.com/u/0/datasources/create?connectorId=AKfycbwhJChhmMypQvNlihgRJMAhCb8gaM3ii9oUNWlW_Cp2PbJSfqeHfPyjNVp15iy9ltCs) lets users create reports and dashboards in Google Data Studio with datasets from InfluxDB v2.
+
+### Add the InfluxDB Connector to Data Studio
+
+1. Add the [InfluxDB Connector data source](https://datastudio.google.com/u/0/datasources/create?connectorId=AKfycbwhJChhmMypQvNlihgRJMAhCb8gaM3ii9oUNWlW_Cp2PbJSfqeHfPyjNVp15iy9ltCs).
```

Comment:
@russorat on the InfluxDB data source page, we say:
_This connector supports one measurement from defined bucket and organization for each data source._

Does this mean when you install the InfluxDB connector (one time), the connector supports one measurement per organization?

Reply:
@noramullen1 @russorat looks like first GDS connector page has changed since yesterday (bucket and measurement aren't available on first page)?

After I enter url, token, and org, then Next, getting this error (and I do have buckets in the org):
![image](https://user-images.githubusercontent.com/47797004/97468298-5c0e4480-1902-11eb-96fa-e3a7c3193e53.png)